### ISSUE TITLE
Better conversion CSS linear-gradient()

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#589]
## Now

Input:

```css
.foo {
  background-image: linear-gradient(to bottom, #f8f8f8, #eeeeee);
}
```

Output:

```css
.foo {
  background-image: -webkit-gradient(linear, left top, left bottom, from(#f8f8f8), to(#eeeeee));
  background-image: -webkit-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: -moz-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: -o-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: linear-gradient(to bottom, #f8f8f8, #eeeeee);
}
```
## But

Input:

```css
.foo {
  background-image: linear-gradient(180deg, #f8f8f8, #eeeeee);
}
```

Output:

```css
.foo {
  background-image: -webkit-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: -moz-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: -o-linear-gradient(top, #f8f8f8, #eeeeee);
  background-image: linear-gradient(to bottom, #f8f8f8, #eeeeee);
}
```
Lost: 

```
  background-image: -webkit-gradient(linear, left top, left bottom, from(#f8f8f8), to(#eeeeee));
```

We need to support 0deg, 90deg,180deg, 270deg, 360deg, converted to WebKit's oldest syntax.

That is to say:

`to bottom = 180deg = top =  left top, left bottom`

> If the argument is to top, to right, to bottom, or to left, the angle of the gradient line is 0deg, 90deg, 180deg, or 270deg, respectively.  
https://drafts.csswg.org/css-images/#linear-gradients

**Comment by ai** [comment]
Done 3f12f38

**Comment by yisibl** [comment]
Also we must be support other [Angle Units](https://drafts.csswg.org/css-values-3/#angle-value)

> For example, a right angle is 90deg or 100grad or 0.25turn or approximately 1.57rad.

**Comment by ai** [comment]
@yisibl ouh :( wait a second.

**Comment by ai** [comment]
@yisibl other angles are finished too 1bbaf47

Wait a second for `-90deg` → `270deg` support.

**Comment by yisibl** [comment]
@ai Nice!

**Comment by ai** [comment]
@yisibl `-90deg` → `270deg` convertation done 0b755a3

**Comment by ai** [comment]
Released in 6.2 version
